### PR TITLE
WIP: packages related to cosmology

### PR DESCRIPTION
**Pull request by rainwoodman**
This is a set of packages we develop / use in cosmology.

This is a WIP.

We are adding in more packages in the coming days.

### CODE REVIEW

File: recipes/bigfile/bld.bat [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"%PYTHON%" setup.py install
```
Comment (synapticarbors) [inline recipes/bigfile/bld.bat]: For simple python installs, you can dispense with the `bld.bat` and `build.sh` and use the `script` section in the recipe yaml file:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L26

File: recipes/bigfile/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
+package:
+  name: bigfile
+  version: "0.1.23"
+
+source:
+  fn: bigfile-0.1.23.tar.gz
+  url: https://pypi.python.org/packages/be/5f/c762c1d543bb3150d5ba32a7669828adf1b6c93106d6ad681b14f981f1f4/bigfile-0.1.23.tar.gz
+  md5: b13f76c3fb10acdc8bef89ab1fa7872a
```
Comment (synapticarbors) [inline recipes/bigfile/meta.yaml]: use sha256

File: recipes/bigfile/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+#  patches:
```
Comment (synapticarbors) [inline recipes/bigfile/meta.yaml]: remove comments

**Comment by rainwoodman** [comment]
Hi @synapticarbors , thanks for looking!

These scripts are created by conda-skeleton, so I am not fully surprised it fails many conda-forge criteria, and I am happy to edit them. 

I have been debating whether to fix conda-verify (e.g. https://github.com/conda/conda-build/issues/1703#issuecomment-275538971 ), and or/ getting conda-skeleton to write files that are directly usable for conda-forge. Then I felt it is too big of a project and want to surrender and do the rest manually from this point. Fair?

Some of these packages tend to be tagged for release very frequently from time to time -- would that be a problem for conda-forge?

Also, I am seeing a strange error on circle-ci -- the builds are crashing with numpy 1.9 and python 3.6 -- any comments on those?

**Comment by synapticarbors** [comment]
It doesn't look like there is a numpy build of 1.9 for python 3.6. It looks like numpy 1.11.2 is the first package with python 3.6 support from looking at `conda search numpy`. 

conda-forge shouldn't have an issue if your packages need to be updated frequently once they are in a feedstock (after they've been accepted from staged-recipes). You would just need to open a pull request for each individual package and update the recipe accordingly then wait for the CI to run and then merge.

**Comment by rainwoodman** [comment]
HI @synapticarbors -- my meta.yaml never mentioned a specific version of numpy though. What shall I do to avoid this issue?

**Comment by rainwoodman** [comment]
We decided to maintain a separate channel because we'd like to pin to the anaconda packages rather than conda-forge packages.  I learned a lot about conda packaging from the conda-forge documentation and @synapticarbors. Thanks for the help!

**Comment by jakirkham** [comment]
FWIW since this PR came in Anaconda 5 came out, which was built on top of conda-forge recipes. Also there has been considerable effort placed on aligning pinnings between `defaults` and `conda-forge`. This aligning of pinnings will be even easier as `conda-build` 3 comes into use in `conda-forge`.

**Pull request closed without merging**